modules/stanford_profile_styles/lib/scss/paragraph-types/wysiwyg/_buttons.scss

```diff
@@ -17,4 +17,14 @@
     @include modular-spacing('margin-bottom', 0);
   }
 
+  // If they're in the same paragraph, get them to stack.
+  [class^="su-button"] {
+    display: block;
+    width: max-content;
+  }
+}
+
+// We want don't want to target the first link.
+[class^="su-button"] + [class^="su-button"] {
```

Comment:
```suggestion
.su-button + .su-button {
```

Reply:
Hi @sherakama,
Thank you for your suggestion. When I tried your approach using
`.su-button + .su-button {`

It rendered as:
Screen Shot 2020-04-21 at 3.38.17 PM

Reply:
With this selector: `[class*="su-button"]` it renders as:
![Screen Shot 2020-04-21 at 3 37 26 PM](https://user-images.githubusercontent.com/284440/79921192-9690a500-83e6-11ea-9878-634a995fd201.png)